documentation/modules/exploit/windows/misc/veeam_one_agent_deserialization.md

```diff
@@ -0,0 +1,120 @@
+## Vulnerable Application
+
+### Description
+
+This module exploits a .NET deserialization vulnerability in Veeam ONE
+Agent before the hotfix versions 9.5.5.4587 and 10.0.1.750.
```

Comment:
Should this be "...in **the** Veeam ONE Agent...". Also can you clarify why there's two versions listed? Is it like 9.5.5.0-9.5.5.4587 and 10.0.1.0 - 10.0.1.750? It's not really clear when this was patched.

Reply:
Sure.